src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -2744,7 +2767,8 @@ var projectUniqueNamesForBuildIntToUpdate
                 packageIdentity: null, // since we have nuGetProjectActions no need packageIdentity
                 primarySources: null, // since we have nuGetProjectActions no need primarySources
                 nuGetProjectContext,
-                token
+                token,
+                versionRange: null
```

Comment:
@zivkan Similar question as before. Should this method be overloaded to accept a `VersionRange` for our public facing API ? 
For now, I only added the parameter to our `internal` method ([here](https://github.com/NuGet/NuGet.Client/pull/3911/files#diff-b441bbfce9030cde6c4e0235943af602fccc9d07cd812a4ea5a9404355a7a856R2788))